data/datasets.md

```diff
@@ -142,6 +142,24 @@ To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the follo
 * `VOC2007_detection` used for evaluation models on VOC2007 dataset for object detection task. Background label + label map with 20 object categories are used. (model examples: [`mobilenet-ssd`](../models/public/mobilenet-ssd/README.md), [`ssd300`](../models/public/ssd300/README.md))
 * `VOC2007_detection_no_bkgr` used for evaluation models on VOC2007 dataset for object detection tasks. Label map with 20 object categories is used.(model examples: [`yolo-v1-tiny-tf`](../models/public/yolo-v1-tiny-tf/README.md))
 
+## [SYGData0829](https://github.com/ermubuzhiming/OMZ-files-download/releases/tag/v1-ly)
+
+### How download dataset
+
+To download SYGData0829 dataset, you need to follow the steps below:
+1. Select ['SYGData0829.rar']https://github.com/legendary111/mobilenet-yolo-syg/releases/download/mobilenet-yolo-syg/SYGData0829.rar
+2. Unpack archive
+
+### Files layout
+* `SYGData0829/dataset_format_VOC2007` - directory containing annotations, images and image sets files directories
+    * `Annotations` - directory containing the SYGData0829 annotation files
+    * `JPEGImages` - directory containing the SYGData0829 images
+    * `ImageSets` - directory containing the SYGData0829 text files specifying lists of images for different tasks
+		* `Main/val.txt` - image sets file for validation of detection tasks
+
+### Datasets in dataset_definitions.yml
+* `SYGData0829` used for evaluation models on SYGData0829 dataset for object detection task. Label map with 4 object categories are used. (model examples: [`mobilenet-yolo-syg`](../models/public/mobilenet-yolo-syg/README.md))
```

Comment:
```suggestion
* `SYGData0829` used for evaluation models on SYGData0829 dataset for object detection task. Label map with 4 object categories are used. (model examples: [`mobilenet-yolo-v4-syg`](../models/public/mobilenet-yolo-v4-syg/README.md))
```
links are tested at precommit, I think this is cause of the precommit fail:
15:50:36  data/datasets.md: URL reference "../models/public/mobilenet-yolo-syg/README.md" target does not exist or is not a file